resources/themes/cura-light/theme.json

```diff
@@ -317,9 +321,11 @@
     },
 
     "sizes": {
+        "base_unit": [1.0, 1.0],
         "window_minimum_size": [70, 50],
         "window_margin": [1.0, 1.0],
         "default_margin": [1.0, 1.0],
+        "double_margin": [2.0, 2.0],
```

Comment:
Rather choose a name like "wide margin". With the name "double margin" it always has to be twice the "default margin", so we might as well just hardcode `default_margin * 2` then. "wide margin" allows for some flexibility.

Reply:
Fair point. Will change.

Reply:
Fixed in 668a929